examples/api/next.config.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
   reactStrictMode: true,
-  transpilePackages: ["@mod-protocol/react"],
+  transpilePackages: [
+    "@mod-protocol/react",
+    // Fixes https://discord.com/channels/896185694857343026/1174716239508156496
+    "@lit-protocol/bls-sdk",
+  ],
```

Comment:
Should this change + lit-related packages in package.json not be in the token gated casts PR?